lib/vet360/models/base_address.rb

```diff
@@ -36,8 +36,12 @@ class BaseAddress < Base
       attribute :created_at, Common::ISO8601Time
       attribute :effective_end_date, Common::ISO8601Time
       attribute :effective_start_date, Common::ISO8601Time
+      attribute :geocode_date, Common::ISO8601Time
+      attribute :geocode_precision, Integer
```

Comment:
how about making this field a string since we receive it as a string from their api and it could contain decimal places?

Reply:
would float be better?

Reply:
float should work too